e2e/utils/pages/activity.page.ts

```diff
@@ -1,21 +1,15 @@
-import type { Page } from '@playwright/test';
-import { ACTIVITY_URL } from '../constants/e2e.constants';
+import { NAVIGATION_ITEM_ACTIVITY } from '$lib/constants/test-ids.constants';
 import { HomepageLoggedIn, type HomepageLoggedInParams } from './homepage.page';
 
 export type ActivityPageParams = HomepageLoggedInParams;
 
 export class ActivityPage extends HomepageLoggedIn {
-	// TODO: Remove this variable, when the activity page is implemented
-	readonly #page: Page;
-
 	constructor({ page, iiPage, viewportSize }: ActivityPageParams) {
 		super({ page, iiPage, viewportSize });
-
-		this.#page = page;
 	}
 
-	// TODO: Implement this method clicking on the navigation item instead of using the URL, when the activity page is implemented
 	override async extendWaitForReady(): Promise<void> {
-		await this.#page.goto(ACTIVITY_URL);
+		await this.clickByTestId(NAVIGATION_ITEM_ACTIVITY);
+		await this.waitForLoadState();
```

Comment:
may you please do this change in another PR?

Reply:
reverted